tests/test_view.py

```diff
@@ -111,190 +103,231 @@ def normalize(self, vec):
+        self.assertEqual(c.up_vector, initial_up_vector)
 
     def test_translation(self):
-        dt = 0.01
-        delta = dt * self.linear_speed()
-        delta_vec = np.array([delta, delta, -delta])
-        new_pos = np.array(self.pos()) + delta_vec
-        new_view_center = np.array(self.view_center()) + delta_vec
+        c = self.camera
+
+        speed = c.linear_speed
+        delta_vec = np.array([speed, speed, -speed])
+        new_position = np.array(c.position) + delta_vec
+        new_view_center = np.array(c.view_center) + delta_vec
 
-        self.move(x=1, dt=dt)
-        self.assertEqual(self.pos()[0], new_pos[0])
+        c.move(x=1)
+        self.assertEqual(c.position[0], new_position[0])
 
-        self.move(y=1, dt=dt)
-        self.assertEqual(self.pos()[1], new_pos[1])
+        c.move(y=1)
+        self.assertEqual(c.position[1], new_position[1])
 
         # camera moves in negative z direction
-        self.move(z=1, dt=dt)
-        self.assertEqual(self.pos()[2], new_pos[2])
+        c.move(z=1)
+        self.assertEqual(c.position[2], new_position[2])
 
         # camera view center should move with camera
-        view_center = self.view_center()
-        self.assertEqual(view_center[0], new_view_center[0])
-        self.assertEqual(view_center[1], new_view_center[1])
-        self.assertEqual(view_center[2], new_view_center[2])
+        self.assertEqual(c.view_center[0], new_view_center[0])
+        self.assertEqual(c.view_center[1], new_view_center[1])
+        self.assertEqual(c.view_center[2], new_view_center[2])
 
     def test_rotation(self):
-        dt = 0.01
-        angle = dt * self.look_speed()
+        c = self.camera
+
+        angle = c.look_speed
 
-        initial_view_center = self.view_center()
-        initial_view_vector = self.view_vector()
+        initial_view_center = c.view_center
+        initial_view_vector = c.view_vector
 
         # test camera does not rotate when left mouse button is not pressed
-        self.move(yaw=1, pitch=1, dt=dt)
-        self.assertEqual(self.view_vector(), initial_view_vector)
-        self.assertEqual(self.view_center(), initial_view_center)
+        c.move(yaw=1, pitch=1)
+        self.assertEqual(c.view_vector, initial_view_vector)
+        self.assertEqual(c.view_center, initial_view_center)
 
         # test camera rotates around y-axis
-        self.move(yaw=1, dt=dt, left_mouse_button=True)
+        c.move(yaw=1, left_mouse_button=True)
 
         rotation_matrix = self.angle_axis(angle, (0, 1, 0))
         rotated_vector = np.array(initial_view_vector) @ rotation_matrix
-        view_vector = self.view_vector()
 
-        self.assertAlmostEqual(rotated_vector[0], view_vector[0], delta=1e-5)
-        self.assertAlmostEqual(rotated_vector[1], view_vector[1], delta=1e-5)
-        self.assertAlmostEqual(rotated_vector[2], view_vector[2], delta=1e-5)
+        self.assertAlmostEqual(rotated_vector[0], c.view_vector[0], places=5)
```

Comment:
`delta` is clearer than `places`.  Why change to the latter?

Reply:
Because flake8's limitation to 80 characters makes me do weird things 😵. `places=5` is two characters less than`delta=1e-5`

Reply:
That's a clear and valid reason.  But it is also unfortunate 🤣.

Could you please rename `rotated_vector` as `rot_vec` or even `rvec`, and change back to use `delta=1.e-5`?  Using `delta` gives us more correct behaviors.  We will copy and paste the code for future testing.  Behaviorally correct code gives us longer mileage.

We cannot change the argument name nor the float literals, but modmesh developers have the constitutive right to rename local variables.  (We have not written how to treat local variable names in the [style guide](https://github.com/solvcon/modmesh/blob/4166c1aa6ffca26f5cd5865606d495d366ddc9e2/STYLE.rst), but we should.)